src/components/CheckList/CheckList.test.tsx

```diff
@@ -214,6 +214,8 @@ test('search matches label name', async () => {
 
 test('clicking label value adds to label filter', async () => {
   renderCheckList();
+  const additionalFilters = await screen.findByText('Additional Filters');
```

Comment:
nit: using `findByText` is fine, but I think `findByRole` is the more encouraged query:

```
findByRole('button', {name: 'Additional Filters', exact: false})
```

Reply:
👍